projects/packages/publicize/package.json

```diff
@@ -26,14 +26,30 @@
 	"browserslist": [
 		"extends @wordpress/browserslist-config"
 	],
+	"exports": {
```

Comment:
I see why you added this. But having a non-js-package as a JS dependency really probably shouldn't work in the first place...

Reply:
Is there a better way to have a PHP package bundle the JS components from `js-packages`?

Reply:
Have the PHP package actually bundle it, instead of being some sort of relay for bundles that actually exist in the plugins using the package.

Reply:
Yeah, I switched to that approach in https://github.com/Automattic/jetpack/pull/38330/commits/311ba28eba6015bed47da8d651db669023ab1acb. Thanks